ambry-api/src/main/java/com.github.ambry/config/CloudConfig.java

```diff
@@ -57,15 +62,39 @@
   @Default(DEFAULT_CLOUD_DESTINATION_FACTORY_CLASS)
   public final String cloudDestinationFactoryClass;
 
+  /**
+   * Require blobs to be encrypted prior to cloud upload?
+   */
+  @Config(VCR_REQUIRE_ENCRYPTION)
+  @Default("false")
+  public final Boolean vcrRequireEncryption;
```

Comment:
any reason for these to be Boolean/Integer instead of boolean/int?

Reply:
not really.  The first field I copied was an Integer but only because it could be null whereas these never are. I changed them to primitives.